Lib/tarfile.py

```diff
@@ -2009,6 +2009,8 @@ def addfile(self, tarinfo, fileobj=None):
                 blocks += 1
             self.offset += blocks * BLOCKSIZE
 
+        self._dbg(3, "%s, size: %i, mtime: %.0f, mode: %i, uname: %s, gname: %s" % (tarinfo.name, tarinfo.size, tarinfo.mtime, tarinfo.mode, tarinfo.uname, tarinfo.gname))
```

Comment:
Please use an f-string and wrap lines at 79 characters.

Reply:
Every other log message in this code uses % interpolation so I just tried to match the style.